test/Makefile.in

```diff
@@ -73,6 +73,9 @@ speculatelog_test: speculatelog_test.c
 lru_test: lru_test.c
 	$(Q)$(CC) -I../src/utils $(CPPFLAGS) $(CFLAGS) -L../src $(LDFLAGS) -o lru_test lru_test.c -lmtev $(LIBMTEV_LIBS)
 
+b64encodev: b64encodev.c
+	$(Q)$(CC) -I../src/utils $(CPPFLAGS) $(CFLAGS) -L../src $(LDFLAGS) -o $@ $< -lmtev $(LIBMTEV_LIBS)
+
```

Comment:
would be nice to just extend the b64 tests in busted that we already have instead of splitting the tests out into C.